presto-yarn-package/src/main/slider/package/templates/queues.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+  "queues": {
+    "global": {
+      "maxConcurrent": 10,

```

Comment:
@FanTn Would you like to do it (in separate patch) once you are working on this?